compiler/passes/normalize.cpp

```diff
@@ -1325,27 +1325,11 @@ static void processSyntacticDistributions(CallExpr* call) {
   if (call->isNamed("chpl__distributed")) {
     if (CallExpr* distCall = toCallExpr(call->get(1))) {
       if (SymExpr* distClass = toSymExpr(distCall->baseExpr)) {
-        if (TypeSymbol* ts = expandTypeAlias(distClass)) {
-          USR_WARN(
+        if (expandTypeAlias(distClass) != nullptr) {
+          USR_FATAL(
             distCall,
-            "omitting 'new' in a dmapped initialization expression is deprecated; please use '<domain> dmapped new <DistName>(<args>)'"
+            "dmapped initialization expression requires a value, not a type - did you mean to use 'new'?"
```

Comment:
I'd keep the wording here that says `<domain> dmapped new <DistName>(<args>)`. "Did you mean to use `new`?" is a bit less informative.